### DISCUSSION TITLE
Front assembly

### DISCUSSION THREAD

**Discussion by XenoProxy** [eusonlito/GPS-Tracker#48] [Q&A] [answered]
I'm running into some frontend trouble. How should I build the project: via gulp or webpack mix? Can anybody explain how to rebuild the front?

**Comment by eusonlito** [comment, marked as answer]
If you want to change styles or javascript you must rebuild the front assets:

```bash
cd resources/views/assets

npm install --legacy-peer-deps

gulp build
```

You can launch `gulp watch` while you are testing changes and then `gulp build` to compile assets.

**Reply by XenoProxy** [reply]
thank you so much!
